programs/stake/src/stake_instruction.rs

```diff
@@ -126,9 +126,12 @@ pub enum StakeInstruction {
 
     /// Set stake lockup
     ///
+    /// If a lockup is not active, the withdraw authority may set a new lockup
+    /// If a lockup is active, the lockup custodian may update the lockup parameters
+    ///
     /// # Account references
     ///   0. [WRITE] Initialized stake account
-    ///   1. [SIGNER] Lockup authority
+    ///   1. [SIGNER] Lockup authority (withdraw authority or lockup custodian)
```

Comment:
Just "or withdraw authority" will do. "lockup custodian" isn't a term used anywhere else and the lockup authority is already mentioned

Reply:
✔️